### PR TITLE
Fe upload profile pic

### PR DESCRIPTION
**Pull request by BrokenRecord85**

### CODE REVIEW

**Review by Connor-Quinny — APPROVED** [review]

File: photoutils.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  return result;
+}
+export async function askForPermission() {
+  const { status } = await ImagePicker.requestCameraPermissionsAsync();
+  return status;
+}
+
+export async function uploadImage(uri, path, fName) {
+    // console.log(path)
+    // console.log(fName)
```
Comment (MartinSWDev) [inline photoutils.js:22]: do we need these ?

File: photoutils.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      };
+      xhr.onerror = function (e) {
+        console.log(e);
+        reject(new TypeError("Network request failed"));
+      };
+      xhr.responseType = "blob";
+      xhr.open("GET", uri, true);
+      xhr.send(null);
+    });
+    //console.log(uri)
```
Comment (MartinSWDev) [inline photoutils.js:38]: or this one

File: photoutils.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    xhr.onload = (event) => {
+      const blob = xhr.response;
+    };
+    xhr.open('GET', url);
+    xhr.send();
+
+    // Or inserted into an <img> element
+    //const img = document.getElementById('myimg');
+   //img.setAttribute('src', url);
+    //})
```
Comment (MartinSWDev) [inline photoutils.js:71]: Whats happening with these comments

File: photoutils.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  // Note that in the URL, characters are URL escaped!
+  const storage = getStorage()
+  const httpsReference = ref(storage, 'gs://dancify-728c9.appspot.com/userPictures/Chad/profilePicture.jpeg');  
+  getDownloadURL(httpsReference)
+  // .then((url) => {
+  //   // `url` is the download URL for 'images/stars.jpg'
+      
+  //     console.log(url, 'Photo utils HEEEEEEEEEEEEEEEEREEEEEEEEEEE') 
+  //  // This can be downloaded directly:
+  //   const xhr = new XMLHttpRequest();
```
Comment (MartinSWDev) [inline photoutils.js:89]: and these

File: photoutils.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  // Create a reference from an HTTPS URL
+  // Note that in the URL, characters are URL escaped!
+  const storage = getStorage()
+  const httpsReference = ref(storage, 'gs://dancify-728c9.appspot.com/userPictures/Chad/profilePicture.jpeg');  
+  getDownloadURL(httpsReference)
+  // .then((url) => {
+  //   // `url` is the download URL for 'images/stars.jpg'
+      
+  //     console.log(url, 'Photo utils HEEEEEEEEEEEEEEEEREEEEEEEEEEE') 
+  //  // This can be downloaded directly:
```
Comment (h-chagas) [inline photoutils.js:88]: Please delete these commented code

File: queryutils.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    querySnapshot.forEach((doc) => {
+      const {email, password, username,} = doc.data()
+      const {id} = doc.id
+    //   console.log(id, email, password, username)
+      newArray.push(doc.data())
+    })
+    // const users = await firestore().collection('users').get()
+    // const docRef = doc(db, "users", "pawel");
+    // const docSnap = await getDoc(docRef);
+
```
Comment (MartinSWDev) [inline queryutils.js:41]: and these

File: src/components/Home.jsx [hunk 7]
```diff
@@ -1,14 +1,57 @@
...
+//   useFetchStreams: false,
+// });
+
+
+// const getIT = async () => {
+//   const q = query(collection(db, 'users'))
+//   const querySnapshot = await getDocs(q)
+//   const newArray = []
+//   querySnapshot.forEach((doc) => {
+//     const {email, password, username,} = doc.data()
```
Comment (MartinSWDev) [inline src/components/Home.jsx:33]: lots here theat need removal

File: queryutils.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+      const {email, password, username,} = doc.data()
+      const {id} = doc.id
+    //   console.log(id, email, password, username)
+      newArray.push(doc.data())
+    })
+    // const users = await firestore().collection('users').get()
+    // const docRef = doc(db, "users", "pawel");
+    // const docSnap = await getDoc(docRef);
+  
+    //   if (docSnap.exists()) {
```
Comment (h-chagas) [inline queryutils.js:42]: Please delete this commented code

File: src/components/UsersList.jsx [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+  
+
+
+  
+
+  
+  //  useEffect(() => {
+    
+  //   getDownloadURL(httpsReference)
+  //   .then((url) => {
```
Comment (MartinSWDev) [inline src/components/UsersList.jsx:37]: and here

**Review by MartinSWDev — APPROVED** [review]

File: src/components/Home.jsx [hunk 10]
```diff
@@ -1,14 +1,57 @@
...
+// const db = initializeFirestore(app, {
+//   experimentalForceLongPolling: true,
+//   useFetchStreams: false,
+// });
+
+
+// const getIT = async () => {
+//   const q = query(collection(db, 'users'))
+//   const querySnapshot = await getDocs(q)
+//   const newArray = []
```
Comment (h-chagas) [inline src/components/Home.jsx:31]: Please delete this commented code

File: src/components/UsersList.jsx [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+    })  
+  }, [])
+  
+
+
+  
+
+  
+  //  useEffect(() => {
+
```
Comment (h-chagas) [inline src/components/UsersList.jsx:35]: Please delete this commented code

**Review by h-chagas — APPROVED** [review]
Please delete commented code where my comments are.

**Pull request merged**